File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    print("Height: ",end= " ")
    height = get_int()
    if height > -1 and height < 24:
        break

for i in range(height):
    for j in range(height-i-1):
        print(" ",end="")
    for k in range(i+1):
        print("#", end="")
    print("  ",end="")
    for l in range(i+1):
        print("#",end="")
    print()

File: pset1/mario/more/mario.c
//pyramid
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int size;
    do
    {
    // get the size of the pyramid
    size = get_int("Height: ");
    }
    // conditions to restric input
    while(size>23 || size<0);

    for(int i=1;i<=size;i++)
    {
        // print spaces
        for(int j=1;j<=size-i;j++)
            printf(" ");
        // print #
        for(int k=1;k<=i;k++)
            printf("#");
        // print in between space of half pyramids
        printf("  ");
        // print right half of the pyramid
        for(int j=1;j<=i;j++)
            printf("#");
        // print next line
        printf("\n");
    }
    // end for of i

}